README.md

```diff
@@ -1,65 +1,277 @@
+### Tool Setup
+Setting up some additional requirements for running OTP4GB-Py can be done with the `setup.py`
+Python script, which is provided with the tool. Running setup is done with the command
+`python setup.py`[^2], which will perform the following steps (some of which require user input).
 
-Populate the `assets` folder with a copy of the OSM map and any GTFS files you want to include in the analysis.
+- Firstly, the script will open a web browser and ask the user to download the Open
+  Street Map (OSM) converter (osmconvert64.exe) and place it in the "bin" directory.
+  The existance of this file in the correct location will be checked before the script
+  moves on.
+- Next the script automatically downloads and installs
+  [Maven](https://maven.apache.org/) into the "bin" folder.
+- Finally, it will use maven to install GTFS filter and OTP.
 
-## Running
+After the setup script has finished the bin folder should contain the following files
+and folders:
 
-You will need to create a directory to set up the tool. This needs the following structure:
+- apache-maven-3.8.6: folder containing the maven installation
+- lib: java dependancies
+- gtfs-filter-0.1.jar: GTFS filter Java executable file
+- otp-2.1.0-shaded.jar: OTP server Java executable file
+- osmconvert64: OSM converter executable file
 
-* `config.yml` A file with config for the run. A sample is provided in the root of this project.
-* `input` directory containing the latest osm download
-* `input\gtfs` directory containing the GTFS files that are to be included in the OTP routing graph
+The final requirement before running the tool is to fill the "assets" directory with
+all required input files. This folder should contain the following:
 
-### `prepare.py`
+- GTFS zip files for all dates, times, modes and areas.
+- Centroids CSV: CSV file containing all the zone centroids for the routing, with the
+  following columns:
+  - zone_id
+  - zone_name
+  - zone_system
+  - latitude
+  - longitude
+- OSM data file for area of interest, information on downloading this file is available
+  on the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Downloading_data).
 
-This script prepares the OTP directory.
+## Running OTP4GB-Py
+This section outlines the scripts involved with running the process and outputting the cost metrics
+and generalised cost matrix. Before running the process a new folder should be created for storing
+the intemediary files and the outputs, this should also contain the OTP4GB-Py configuration file.
 
-Usage:
+The config file should be named "config.yml" and uses the
+[YAML format](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-syntax),
+an [example config](example-config.yml) is provided and the parameters for the it are outlined
+in table 1.
 
+**Table 1:** *Details the parameters available in the config file and their default values.*
```

Comment:
Newline required between heading & table [for correct rendering](https://stackoverflow.com/a/46697788) - See L181 above Table 2

Reply:
Fixed in 73014cf3fff0ce7ecfc1b0144123aafe469d019c